### PR TITLE
feat: v6.3 upgrade handler and release notes

### PR DESCRIPTION
**Pull request by robert-zaremba**
## Summary by CodeRabbit

- **New Features**
	- Added symbol name to the x/uibc QueryAllOutflowsResponse `outflows` entry.
	- Added new queries `uibc/MissCounters` and `uibc/Inflows`.
	- Addition of query `leverage/RegisteredTokenMarkets` to fetch Registered Tokens and their Market Summaries for frontend in fewer queries.

- **Improvements**
	- Modification of parameters in `uic/MsgGovUpdateQuota` to `uibc/MsgGovUpdateQuota`.
	- Addition of ICS 20 memo handler (disabled) in UIBC.
	- Adjusted IBC Quota v2 mechanism for better quota management.

- **Bug Fixes**
	- Fix in inflow amount calculation.

- **Documentation**
	- Updated CHANGELOG to reflect the latest changes and improvements.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -6,43 +6,20 @@
...
-  We set `MinInitialDepositRatio` to 10%.`
-- Added `meToken` WASM queries.
-- IBC Quota v2 mechanism
-  1. Outflows quota has been increased to `$1.6M` for total outflows and `$1.2M` per token outflows.
-  2. new lifting conditions is added: IBC outflows are possible if (1.) fails, but
-     - sum outflows of all tokens <= `$1M +  InflowOutflowQuotaRate * sum_of_all_inflows`;
-     - and token outflows <= `$0.9M + InflowOutflowQuotaRate * token_inflows`.
-       See [IBC Quota Design](./x/uibc/README.md#design) for more details.
+- Cosmos SDK v0.47.7 patch update.
+- New queries: `uibc/MissCounters`, `uibc/Inflows`, `leverage/RegisteredTokenMarkets`
```
Comment (toteki) [inline RELEASE_NOTES.md:14]: Is `MissCounters` part of uibc?

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -6,43 +6,20 @@
...
+- Update `uic/MsgGovUpdateQuota` Msg type to handle the new inflow parameters.
```
Comment (toteki) [inline RELEASE_NOTES.md:15]:
```suggestion
- Update `uibc/MsgGovUpdateQuota` Msg type to handle the new inflow parameters.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -46,23 +46,25 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
+- [2370](https://github.com/umee-network/umee/pull/2370) Add missing params to `uibc/MsgGovUpdateQuota`.
 - [2374](https://github.com/umee-network/umee/pull/2374) Add symbol name to the x/uibc QueryAllOutflowsResponse `outflows` entry.
 
 ### Features
 
 - [2352](https://github.com/umee-network/umee/pull/2352) new `uibc/MissCounters` query
 - [2352](https://github.com/umee-network/umee/pull/2352) new `uibc/Inflows` query.
-- [2349](https://github.com/umee-network/umee/pull/2349) UIBC: adding ICS 20 memo handler.
+- [2349](https://github.com/umee-network/umee/pull/2349) UIBC: adding ICS 20 memo handler (disabled).
 - [2369](https://github.com/umee-network/umee/pull/2369) Add query `leverage/RegisteredTokenMarkets` to fetch Registered Tokens and their Market Summaries for frontend in fewer queries.
```
Comment (toteki) [inline CHANGELOG.md:62]:
```suggestion
- [2369](https://github.com/umee-network/umee/pull/2369) Add query `leverage/RegisteredTokensWithMarkets` to fetch Registered Tokens and their Market Summaries for frontend in fewer queries.
```
To match name change in PR

**Review by toteki — APPROVED** [review]
approved w/comments

File: RELEASE_NOTES.md (hunk 1 above)
Comment (gsk967) [inline RELEASE_NOTES.md:14]:
```suggestion
- New queries: `oracle/MissCounters`, `uibc/Inflows`, `uibc/QuotaExpires`,`leverage/RegisteredTokenMarkets`
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -46,23 +46,25 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
 ### Improvements
 
 - [2363](https://github.com/umee-network/umee/pull/2363) Upgrade Cosmos SDK to v0.47.7.
-- [2370](https://github.com/umee-network/umee/pull/2370) Add missing params to `uic/MsgGovUpdateQuota`.
+- [2370](https://github.com/umee-network/umee/pull/2370) Add missing params to `uibc/MsgGovUpdateQuota`.
 - [2374](https://github.com/umee-network/umee/pull/2374) Add symbol name to the x/uibc QueryAllOutflowsResponse `outflows` entry.
 
 ### Features
 
 - [2352](https://github.com/umee-network/umee/pull/2352) new `uibc/MissCounters` query
```
Comment (gsk967) [inline CHANGELOG.md:59]:
```suggestion
- [2352](https://github.com/umee-network/umee/pull/2352) new `oracle/MissCounters` query
```

**Review by gsk967 — APPROVED** [review]
Pre-Approving

File: RELEASE_NOTES.md (hunk 1 above)
Comment (robert-zaremba) [reply RELEASE_NOTES.md:14]: no, it's part of the oracle.  Thanks!

**Pull request merged**